shared/src/main/java/com/couchbase/lite/ValueIndexItem.java

```diff
@@ -32,8 +34,13 @@ private ValueIndexItem(Expression expression) {
      *
      * @param property the property name
      * @return The value index item
+     * @throws IllegalArgumentException when property is null.
      */
-    public static ValueIndexItem property(String property) {
+    public static ValueIndexItem property(@NonNull String property) {
+
+        if(property == null) {
+            throw new IllegalArgumentException("property is null");
+        }
```

Comment:
Summary:

1. Removed `@throws IllegalArgumentException when alias is null.` as it is not required to declare throwing a runtime exception. We haven't declared this in other places e.g. Database class as well.

2. Remove an extra empty line before `if(<variable> == null) {`.

3. Add a space between if and (. Should be `if (<variable> == null)`.

4. Add . (full stop) at the end of "... is null" -> `... is null.`

5. Do the same for the other methods.